emitter/__init__.py

```diff
@@ -59,7 +58,14 @@ def Emit(self, *args):
         rooms = self._getRooms()
         extras['rooms'] = rooms if len(rooms) > 0 else ''
 
-        self._client.publish(self._key, msgpack.packb([packet, extras]))
+        self._client.publish(self._key + "#emitter", msgpack.packb([packet, extras]))
```

Comment:
Send duplicate indeed not a good idea.
But if socket.io-redis < 0.2  not will receive TWO messages。 See # 11

socket.io change protocol is a hassle.